recipes/pymc3/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pymc3" %}
+{% set version = "3.0rc2" %}
+{% set md5 = "80105941758b9eba47c8a2774d5ee905" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python >=3.4
```

Comment:
@ericmjl pymc3 is Python 3 only, right? You need to remove the Python restriction here and add (`skip: True  # [py2k]`) to the build section like:

```yaml
build:
   number: 0
   skip: True  # [py2k]
   script: python setup.py install --single-version-externally-managed --record record.txt
```

that will help `conda-build-all` to do the right thing.